packages/diff-sequences/src/__tests__/index.property.test.ts

```diff
@@ -0,0 +1,82 @@
+import diff from '../';
+import fc from 'fast-check'
+
+const findCommonItems = (a: string[], b: string[]): string[] => {

```

Comment:
`lint-and-typecheck` failed on CI with `Array type using 'string[]' is forbidden. Use 'Array<string>' instead.`

Reply:
Will do

Reply:
Thank you. It looks like a few more lint errors. Does `yarn lint --fix` locally display them?

Reply:
I thought I did, I'll retry

Reply:
@pedrottimark This time I did it for sure ^^ I might not be at head. Maybe I could just merge onto master on branch to get the latest changes

Reply:
Ohhh, now it’s about the Facebook comment at the beginning of the file. I always forget it.

You can copy and paste from `index.test.ts` file

Reply:
Got it: `Facebook copyright header check failed for the following files`